src/runtimes/node/utils/node_version.ts

```diff
@@ -14,6 +15,7 @@ export const getNodeSupportMatrix = (configVersion?: string): NodeVersionSupport
 
   return {
     esm: versionNumber >= 14,
+    awsSDKV3: versionNumber >= 18,
```

Comment:
It feels a bit weird to me to be using the Node support matrix to signal whether we need to bundle a specific version of the AWS SDK, because that has nothing to do with Node itself, just a Lambda implementation detail.

Reply:
Yeah, I get that - but at the other hand, it is a Node.js-only issue, and it's totally dependent on the version of Node.js. That's why I put it into here. But if you prefer putting it into a different function name, I can refactor it 🤷

Reply:
Up to you. If you think this simplifies things, let's keep it.

Regardless, it feels that making this a `number` rather than `boolean` (e.g. `awsSDKVersion: 3`) would give us a bit more flexibility in case Lambda ships a future version of Node with yet another major version of their SDK? With the current implementation we'd need to keep adding more properties and end up with a potential inconsistent state?

Reply:
Agree, but let's make that update when AWS ships the next version.